content/docs/09.administrator-guide/03.monitoring/index.md

```diff
@@ -1,23 +1,90 @@
 ---
-title: Monitoring & Alerting
+title: Alerting & Monitoring
 ---
 
-Kestra will deploy a monitoring endpoint on port 8081 by default. (You can change this port with
-[configuration options](../01.configuration/index.md) `endpoints.all.port`)
+## Alerting
+
+Failure alerts are non-negotiable. When a production workflow fails, you should get notified about it as soon as possible. To implement failure alerting, you can leverage Kestra's built in notification tasks, including: 
+- [Slack](../../../plugins/plugin-notifications/tasks/slack/io.kestra.plugin.notifications.slack.slackexecution.md) 
+- [Microsoft Teams](../../../plugins/plugin-notifications/tasks/teams/io.kestra.plugin.notifications.teams.teamsexecution.md) 
+- [Email](../../../plugins/plugin-notifications/tasks/mail/io.kestra.plugin.notifications.mail.mailexecution.md)
+
+
+Technically, you can add custom failure alerts to each flow separately using the [`errors` tasks](../../02.tutorial/06.errors.md):
+
+```yaml
+id: onFailureAlert
+namespace: blueprint
+
+tasks:
+  - id: fail
+    type: io.kestra.plugin.scripts.shell.Commands
+    runner: PROCESS
+    commands:
+      - exit 1
+
+errors:
+  - id: slack
+    type: io.kestra.plugin.notifications.slack.SlackIncomingWebhook
```

Comment:
Better to use a SlackExecution and explain that the template can be customized by using a SlackIncommingWebhook.
Each notification plugin comes with 2 tasks, one with an execution message (meaning a message that includes the information of the status of an execution) and one where the message can be customized.

Reply:
yup, the SlackExecution only works in triggers and listeners, that's why here we use IncomingWebhook. It doesn't work on `errors`. Please share a flow example if this is not true. 

Regarding:
> On the Alerting and Monitoring page, one day we should provide a "Sanity Check" flow and explain that Kestra can be monitored externally by calling periodically this flow (for ex with a webhook).

do you want to open a dedicated issue? I was convinced that such monitoring flow is not needed because of the already existing healthcheck endpoint http://localhost:8081/health

Reply:
> SlackExecution only works in triggers and listeners

OK then.

> do you want to open a dedicated issue? I was convinced that such monitoring flow is not needed because of the already existing healthcheck endpoint http://localhost:8081/health

It tests more than just "the port is open", it can be a complete sanity check that checks DB connectivity, queue functionality, internal storage, ... I'll open an issue for that.

Reply:
ahh I see, that makes sense, thanks so much!